src/Tribe/Ticket_Object.php

```diff
@@ -507,11 +516,22 @@ public function is_in_stock() {
 				return true;
 			}
 
-			$remaining = $this->inventory();
+			/* @var Tribe__Cache $cache */
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 4c4356cbdc40a2da8ee707314028000e</sub>